src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -79,225 +123,455 @@ const useStyles = makeStyles()(() => ({
+                            flexDirection="column"
+                          >
+                            <Box>
+                              <Typography
+                                fontWeight={700}
+                                data-testid="donationDate"
+                              >
+                                {DateTime.fromISO(
+                                  donation.amount.conversionDate,
+                                )
+                                  .setLocale(locale || 'en')
+                                  .toLocaleString()}
+                              </Typography>
+                            </Box>
+                            <Box>
+                              <Typography
+                                sx={{ textAlign: 'center' }}
+                                data-testid="donationAmount"
+                              >
+                                {`${donation.amount.amount} ${donation.amount.currency}`}
+                              </Typography>
+                            </Box>
+                          </Grid>
+                        ))}
+                      </Grid>
+                    )}
+                  </Grid>
+                </Card>
+                <Grid item xs={12} md={3} className={classes.buttonGroup}>
+                  <Box className={classes.buttonGroupBox}>
+                    <Box className={classes.buttonTop}>
+                      <Button
+                        type="submit"
+                        variant="contained"
+                        data-testid="confirmButton"
+                        style={{ width: '100%' }}
+                      >
+                        {t('Confirm')}
+                      </Button>
+                    </Box>
+                    <Box className={classes.buttonBottom}>
+                      <Button
+                        variant="contained"
+                        style={{ width: '100%' }}
+                        data-testid="doNotChangeButton"
+                        onClick={() =>
+                          showModal(
+                            modalContact,
+                            t(
+                              `Are you sure you wish to leave {{source}}'s commitment information unchanged?`,
+                              { source: name },
+                            ),
+                            t("Don't Change"),
+                            UpdateTypeEnum.DontChange,
+                          )
+                        }
+                      >
+                        {"Don't Change"}
+                      </Button>
+                    </Box>
+                    <Box className={classes.ButtonIcons}>
+                      <IconButton
+                        data-testid="goToContactsButton"
+                        onClick={() => setContactFocus(id, 'Donations')}
+                      >
+                        <SearchIcon />
+                      </IconButton>
```

Comment:
```suggestion
```

Reply:
Now that contacts can pop out on the right sidebar I don't think we need this button. On the new MPDX users should be used to clicking on the name of the contact to see the contact panel.